webview/platforms/cocoa.py

```diff
@@ -433,6 +434,11 @@ def first_show(self):
             self._add_view_menu()
 
             BrowserView.app.activateIgnoringOtherApps_(Foundation.YES)
+            if _multiprocessing:
```

Comment:
i have no MacOS computer, so you and other contributors should add multiprocessing to macos.
use this lines to add multiprocessing to macos cocoa